metadata.yaml

```diff
@@ -4,6 +4,10 @@ name: dex-auth
 summary: A federated OpenID Connect provider
 description: |
   Dex is an identity service that uses OpenID Connect to drive authentication for other apps.
+website: https://charmhub.io/dex-auth
+source: https://github.com/canonical/dex-auth-operator
+issues: https://github.com/canonical/dex-auth-operator/issues
+docs: https://discourse.charmhub.io/t/charmed-dex/8214
```

Comment:
do we have a page on https://charmed-kubeflow.io ?

Reply:
Uh, if you mean something like https://charmed-kubeflow.io//dex-auth, no we do not. We have a list of all the components [here](https://charmed-kubeflow.io/docs/kubeflow-bundle#heading--components), but I don't think that links belongs in here.